src/main/java/org/terasology/irlCorp/systems/MultiBlockTankAuthoritySystem.java

```diff
@@ -66,10 +66,10 @@ public void initialise() {
 
                     @Override
                     public void multiBlockFormed(BlockRegion region, EntityRef entity, Void designDetails) {
-                        double bonusFromFootprint = Math.pow(1.2, Math.min(region.getSizeX(), region.getSizeZ() * 0.5));
+                        double bonusFromFootprint = Math.pow(1.2, Math.min(region.sizeX(), region.sizeZ() * 0.5));
                         FluidInventoryComponent fluidInventoryComponent = new FluidInventoryComponent(
                                 1,
-                                (float) (region.getSizeX() * region.getSizeY() * region.getSizeZ() * 10000 * bonusFromFootprint));
+                                (float) (region.sizeX() * region.sizeY() * region.sizeZ() * 10000 * bonusFromFootprint));
```

Comment:
I'm wondering whether this should be offered as `region.size()` (or `region.volume()` or `region.blockCount()` or whatever...)